src/freenas/etc/rc.conf.local

```diff
@@ -576,15 +590,17 @@ _gen_conf()
 	while read -r nonroot mountdport statdport lockdport; do
 		if [ "${nonroot}" = 1 ]; then
 			mountdflags="${mountdflags}n"
+                        statdflags="${statdflags}n"
+                        lockdflags="${lockdflags}n"
 		fi
```

Comment:
These lines do not seem right.
e.g. if lockdflags=" -d 10" then lockdflags="${lockdflags}n" will become lockdflags=" -d 10n" and that is not a valid option.